### DISCUSSION TITLE
File Upload component getElement not working

### DISCUSSION THREAD

**Discussion by tom-ramesh** [primefaces/community#1822] [PrimeReact] [answered]
I'm trying to access inner elements by getting the container element but it throws an error. Is there something I'm doing wrong?

![image](https://github.com/primefaces/community/assets/88586002/18e685a8-2497-44dc-af9f-48e9a628f591)

![image](https://github.com/primefaces/community/assets/88586002/c0ea18cf-fc02-4091-a41b-13b9e677e4aa)

**Comment by nitrogenous** [comment]
Could you please provide a StackBlitz example demonstrating your approach?

**Reply by tom-ramesh** [reply]
Hi @nitrogenous , sharing the [StackBlitz ](https://stackblitz.com/edit/ragzir?file=src%2FFileUpload.tsx)example, 

I want the error msg to be gone when the user try to drop another file, that's what I'm trying to achieve here.

**Reply by tom-ramesh** [reply]
Hi, any idea on how to het the input element ref so that I can prepopulate the value with a file(ObjectURL) during the edit phase of a form.

**Reply by tom-ramesh** [reply, marked as answer]
got it working use this line of code
```
fileUploadRef.current.setFiles([presetFile])
```
